lang/en.js

```diff
@@ -475,6 +475,7 @@ export default {
   },
   "items": {
     "itemCount": "1 item |{count} items",
+    "itemOf": "100 of {count} items",
```

Comment:
100 should be a variable inserted by interpolation, not hard-coded here.

Reply:
ok 100 has been moved to a variable